vars/runBenchmarks.groovy

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env groovy
+
+/**
+ * Run JMH benchmarks and archives results
+ * @param artifacts archive these JMH reports
+ * @since TODO
+ */
+def call(String artifacts = null) {
+    // TODO: Use Lockable Resources plugin
```

Comment:
I am not sure we can merge it without it. Maybe it makes sense to create an INFRA ticket to @olblak to get it set up. Maybe 2 parallel runs is a good limit for now.

Reply:
Created https://issues.jenkins-ci.org/browse/INFRA-2141

Reply:
the lockable plugin is already installed

Reply:
@olblak We need you to create a resource that we can use for throttling benchmarks running on `highmem` nodes.

Reply:
so something like lock('runBenchmarks') {
}

should be fine